conda/recipes/librmm/meta.yaml

```diff
@@ -60,12 +67,19 @@ outputs:
       run_exports:
         - {{ pin_subpackage("librmm", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
+        {% endif %}
     requirements:
       build:
         - cmake {{ cmake_version }}
       run:
-        - cudatoolkit {{ cuda_spec }}
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% endif %}
```

Comment:
This is worth investigating but I'd like to avoid changing the CUDA 11 packaging in this PR. Let's leave `cudatoolkit` as a run dependency for CUDA 11 packages for now.

There are potential unforeseen consequences. I think this could affect users mixing `conda-forge` and `nvidia` channels for CUDA 11, for instance.